anime-relations.txt

```diff
@@ -154,9 +169,15 @@
 # Date A Live II -> Date A Live: Encore OVA
 - 19163|7864:11 -> 22961|8511:1
 
+# Deadman Wonderland -> ~ OVA
+- 6880|4765:13 -> 10418|6172:1
+
 # Denpa Onna to Seishun Otoko -> ~ Special
 - 9379|5710:13 -> 10638|6276:1
 
+# Detroit Metal City -> ~: Birth of the Metal Devil
+- 3702|3202:13 -> 5065|3911:1
```

Comment:
"Birth of the Metal Devil" is just a summary of the TV series, so it shouldn't be the 13th episode.